akka-persistence-typed/src/main/scala/akka/persistence/typed/internal/Running.scala

```diff
@@ -283,6 +283,17 @@ private[akka] object Running {
           // ignore
           this // it will be stopped by the first WriteMessageFailure message; not applying side effects
 
+        case DeleteMessagesSuccess(toSequenceNr) ⇒
+          setup.log.debug("PersistentShard messages to [{}] deleted successfully.", toSequenceNr)
+          internalDeleteSnapshots(toSequenceNr)
+          this
+
+        case DeleteMessagesFailure(e, toSequenceNr) ⇒
+          setup.log.warning(
+            "Failed to delete messages toSequenceNr [{}] for persistenceId [{}] due to [{}: {}].",
+            toSequenceNr, setup.persistenceId.id, e.getClass.getCanonicalName, e.getMessage)
+          this
```

Comment:
new Behavior? this?

Reply:
that is ok, failure to delete isn't fatal and we can just continue